docs/contributing/feature-flags.md

```diff
@@ -97,6 +92,107 @@ configuration setting can be used to override it. The file must be present befor
 solution, but once there you can change the file contents and see immediate results in running /
 debugging code.
 
+<details>
+  <summary>Context-aware feature flag JSON</summary>
+  <div>
+    <div>
+      The `flags.json` file can also define flags which respond to user context. Currently, only `UserId`and the `OrganizationId` of all organizations a user belongs to are included in our feature flagging context. The syntax for defining context-aware flags amounts to defining a `flag` object which specifies `variations` values and `rules` which are evaluated against the user context. A `fallthrough` object is also available to specify a default variation. Rules are represented by `segments`, which are defined in the same file.
```

Comment:
```suggestion
      The `flags.json` file can also define flags that respond to user context. Currently, only `UserId` and the `OrganizationId` of all organizations a user belongs to are included in our feature flagging context. The syntax for defining context-aware flags amounts to defining a `flag` object that specifies `variations` values and `rules` which are then evaluated against the user context. A `fallthrough` object is also available to specify a default variation. Rules are represented by `segments`, which are defined in the same file.
```

Reply:
⛏️ This is also violating our line lengths since it's in custom HTML.

Reply:
❌ We support an addition service account context as well that's omitted.

Reply:
Good point, we actually have three `contextKinds` now that I look closer. I'll expand to include descriptions of data on each